src/server/scripts/EasternKingdoms/ZulAman/zulaman.cpp

```diff
@@ -52,56 +128,225 @@ class npc_forest_frog : public CreatureScript
+                                case NPC_GUNTER:    Talk(SAY_GUNTER_2, player); break;
+                                case NPC_KYREN:     Talk(SAY_KYREN_2, player); break;
+                                case NPC_MITZI:     Talk(SAY_MITZI_2, player); break;
+                                case NPC_CHRISTIAN: Talk(SAY_CHRISTIAN_2, player); break;
+                                case NPC_BRENNAN:   Talk(SAY_BRENNAN_2, player); break;
+                                case NPC_HOLLEE:    Talk(SAY_HOLLEE_2, player); break;
+                            }
+                            eventTimer = 4;
+                            if (me->GetEntry() == NPC_GUNTER || me->GetEntry() == NPC_KYREN)
+                                events.ScheduleEvent(eventTimer, 5*MINUTE*IN_MILLISECONDS); // vendors wait for 5 minutes before running away and despawning
+                            else
+                                events.ScheduleEvent(eventTimer, 6000);
+                            break;
+                        case 4:
+                            me->HandleEmoteCommand(EMOTE_ONESHOT_WAVE);
+                            switch (me->GetEntry())
+                            {
+                                case NPC_MANNUTH:   Talk(SAY_MANNUTH_3, player); break;
+                                case NPC_DEEZ:      Talk(SAY_DEEZ_3, player); break;
+                                case NPC_GALATHRYN: Talk(SAY_GALATHRYN_3, player); break;
+                                case NPC_ADARRAH:   Talk(SAY_ADARRAH_4, player); break;
+                                case NPC_DARWEN:    Talk(SAY_DARWEN_3, player); break;
+                                case NPC_FUDGERICK: Talk(SAY_FUDGERICK_3, player); break;
+                                case NPC_MITZI:     Talk(SAY_MITZI_3, player); break;
+                                case NPC_CHRISTIAN: Talk(SAY_CHRISTIAN_3, player); break;
+                                case NPC_BRENNAN:   Talk(SAY_BRENNAN_3, player); break;
+                                case NPC_HOLLEE:    Talk(SAY_HOLLEE_3, player); break;
+                            }
+                            eventTimer = 5;
+                            events.ScheduleEvent(eventTimer, 2000);
+                            break;
+                        case 5:
+                            switch (me->GetEntry())
+                            {
+                                case NPC_ADARRAH:
+                                    DoCast(me, SPELL_STEALTH_, true);
+                                    break;
+                            }
+                            if (me->GetPositionY() > 1290.0f)
+                                me->GetMotionMaster()->MovePoint(POINT_DESPAWN, 118.2742f, 1400.657f, -9.118711f);
+                            else
+                                me->GetMotionMaster()->MovePoint(POINT_DESPAWN, 114.3155f, 1244.244f, -20.97606f);
+                            eventTimer = 0;
+                            break;
+						default:
```

Comment:
Tabs here

Reply:
Please replace the tabs before "default:" with spaces.